packages/yoroi-extension/features/step_definitions/common-steps.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import { Before, BeforeAll, Given, Then, After, AfterAll, setDefinitionFunctionWrapper, setDefaultTimeout } from 'cucumber';
+import { Before, BeforeAll, Given, Then, After, AfterAll, setDefinitionFunctionWrapper, setDefaultTimeout, Status,setWorldConstructor} from 'cucumber';
```

Comment:
Please, try to follow the project code style. I mean spaces.

Reply:
You can download `eslint` extenstion